src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    const resultKey = Object.keys(result);
+
+    if (resultKey.includes(robot[key])) {
+      return null;
+    }
```

Comment:
This can be simplified.
```suggestion
    if (robot[key] in result) {
      return null;
    }
```